src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -178,7 +178,7 @@ bool MulticopterLandDetector::_get_maybe_landed_state()
 	const hrt_abstime now = hrt_absolute_time();
 
 	// When not armed, consider to be maybe-landed
-	if (!_actuator_armed.armed || (_vehicle_attitude.timestamp == 0)) {
```

Comment:
`_vehicle_attitude` is now unused and can be removed completely

Reply:
> `_vehicle_attitude` is now unused and can be removed completely

Thanks @bkueng !  Fixed with #13134